solr/core/src/resources/security.json

```diff
@@ -0,0 +1,74 @@
+{
+  "authentication": {
+    "blockUnknown": false,
+    "class": "solr.BasicAuthPlugin",
```

Comment:
What about type=kerberos?

Reply:
Humm..  My somewhat snarky response is "What is this Kerberos thing you speak of"?   So, the auth tool didn't support that previously, so I never thought about it.   Maybe for a seperate ticket?   I wonder if we need to support "point at a existing security.json"?   Or, do we add another param to the command?

Reply:
Never used it. Looks like you'll configure your entire KRB environment in other files and sysprops, and then at the end you create security.json with little more than the "class" property. When creating user:role mapping, the username needs to match the kerberos principal, so I suppose that a user could be mapped to role here too, although it would not be **add**user, but perhaps **map**user?

Probably best to document that for kerberos we only support initial bootstrap of security.json, no user:role mapping, which can be done in the UI (?)

Reply:
Since we know so little about KRB, we already have the text "Currently, this script only enables Basic Authentication, and is only available when using SolrCloud mode" in the ref guide, which I think covers the concern.  I don't know anything about KRB, and don't think I ever will!

Reply:
wait, we have a `-type` command...  so yes, we do need to update the ref guide...